### ISSUE TITLE
binders designed don't binds at the hotspots of the target chain.

### ISSUE THREAD

**Issue by rocke2020** [sokrypton/ColabDesign#172]
run file: af/examples/peptide_binder_design.ipynb
hope: the binder binds at the hotspot, e.g. 32-41 index in the target chain.
results: the binder not binds at the hotspots at the target chain at all. 

configs (use 1YCR because it is simple): 
pdb = "1YCR"
target_hotspot: "32-41"
target_chain: "A"
target_flexible: false
binder_chain = ""
binder_len: 12
binder_seq: ""
num_recycles: 3
recycle_mode: "sample"
num_models: 5
use_multimer: True
optimizer_protocol: "pssm_semigreedy"
soft_iters: 150
hard_iters: 20
gd_method: "sgd"
learning_rate: 0.05
norm_seq_grad: True
dropout: True
